### PR TITLE
Replaced duplicate v2.3 file with symlink and cleaned services topics

### LINKED ISSUE #5499: Feedback on page: /guides/v2.3/cloud/project/project-conf-files_services-rabbit.html
# General issue

## Description:

Procedure for enabling Rabbitmq on Cloud does not include step to verify the configuration after enabling.

## Possible solutions:
It would be useful if the section on enabling RabbitMq topic had a procedure with instructions similar to the ones in the [Enable My SQL](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-mysql.html#enable-mysql) topic which provides clearer instructions and also includes the step to validate services. 
- Change the [Add RabbitMQ in services.yaml and .magento.app.yaml](https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-rabbit.html#settings) title to "Enable RabbitMQ".
- Reformat procedure steps so they are similar to the ones in the Set up MySQL topic.
- Add the step to verify service relationships.

### PR DESCRIPTION
**Pull request by meker12**
Fixes #5499 
# Purpose of this pull request

-  Internal versioning project: Replaced version v2.3 of project-conf-files_services-elastic.md file with a symlink to the v2.2 topic
-  Updated the Enable RabbitMQ service instructions to match the other service-enablement procedures and added missing step to verify service relationships
- Added alternative pretty-print command to instructions for listing MAGENTO_CLOUD_RELATIONSHIPS information
- Cleaned the Set up Elastic search topic to place admonitions more strategically and link to the service-versions topic.
- Removed specific version from enable service procedures to simplify topic maintenance
- Fixed ul and ol spacing to pass lint checks

### To do
- [x] Preview build (https://devdocs.magedevteam.com/909/guides/v2.3/cloud/project/project-conf-files_services.html)
- [ ] Editorial review

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/project/project-conf-files_services-mysql.html
- https://devdocs.magento.com/guides/v2.3/project/project-conf-files_services-rabbit.html
- https://devdocs.magento.com/guides/v2.3/project/project-conf-files_services-redis.html
- https://devdocs.magento.com/guides/v2.3/project/project-conf-files_services.html
- https://devdocs.magento.com/guides/v2.3/project/project-integrate-blackfire.html

### CODE REVIEW

File: guides/v2.2/cloud/project/project-conf-files_services-rabbit.md [hunk 1]
```diff
@@ -10,31 +10,38 @@ The [Message Queue Framework (MQF)]({{ page.baseurl }}/config-guide/mq/rabbitmq-
...
 If you prefer using an existing AMQP-based service, like RabbitMQ, instead of relying on {{site.data.var.ece}} to create it for you, use the [`QUEUE_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#queue_configuration) environment variable to connect it to your site.
 
 {% include cloud/service-config-integration-starter.md %}
 
-## Add RabbitMQ in services.yaml and .magento.app.yaml {#settings}
+{:.procedure}
+To enable RabbitMQ:
 
-To enable RabbitMQ, add the following code with your installed version and allocated disk space in MB to the `.magento/services.yaml` file.
+1. Add the required name, type, and version to the `.magento/services.yaml` file along with the the installed RabbitMQ version and allocated disk space in MB:
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/project-conf-files_services-rabbit.md]: Technically, version is not a service value as defined in the main services topic. It is more a value of a value.

```suggestion
1. Add the required name, type, and disk value (in MB) to the `.magento/services.yaml` file along with the the installed RabbitMQ version.
```

File: guides/v2.2/cloud/project/project-conf-files_services-redis.md [hunk 2]
```diff
@@ -16,18 +16,18 @@ We support Redis versions 3.2 and 5.0. See [Configure Redis]({{ page.baseurl }}/
 {:.procedure}
 To enable Redis:
 
-1. Add the required name and type to the `.magento/services.yaml` file.
+1. Add the required name, type, and version to the `.magento/services.yaml` file.
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/project-conf-files_services-redis.md]: Per the main services topic, the two values required are `name` and `type`. The name value is the relationship used in configuration file and can be anything you want as long as it is consistently used. The type value actually includes two values: service name and service version. In the SQL topic, we say add the name, type, and disk value. These are meant to refer to service values defined on the main services topic.

Comment (meker12) [inline guides/v2.2/cloud/project/project-conf-files_services-redis.md]:
```suggestion
1. Add the required name and type to the `.magento/services.yaml` file.
```

File: guides/v2.2/cloud/project/project-conf-files_services.md [hunk 3]
```diff
@@ -101,53 +101,60 @@ The current default storage amount per project is 5GB, or 5120MB. You can distri
 
 ## Service relationships
 
-{{site.data.var.ece}} uses the [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{page.baseurl}}/cloud/env/environment-vars_cloud.html) environment variable to retrieve environment-related relationships. For services to be available to an application in your project, you must specify [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) between applications and services in the `.magento.app.yaml` file.
+In {{ site.data.var.ece }} projects, service [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) configured in the `.magento.app.yaml` file determine which services are available to your application. The service must be configured on your {{ site.data.var.ece }} infrastructure and enabled in the [services.yaml]({{ page.baseurl}}/cloud/project/project-conf-files_services.html) file.
```
[the remark this answers is not in the document]
Comment (meker12) [reply guides/v2.2/cloud/project/project-conf-files_services.md]: I was trying to convey that available services require two things:  configure relationships, added to services.yaml.  I think I don't need to state that here, and can just remove the second sentence.

Comment (meker12) [inline guides/v2.2/cloud/project/project-conf-files_services.md]:
```suggestion
In {{ site.data.var.ece }} projects, service [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) configured in the `.magento.app.yaml` file determine which services are available to your application.
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**